### DISCUSSION TITLE
Can i join team?sir?

### DISCUSSION THREAD

**Discussion by mozzaru** [ROM-builders/temporary#18367] [General]
Git username : mozzaru
Dt: https://github.com/mozzaru/android_device_xiaomi_markw/tree/derpfest-13

Successful Build:
https://cirrus-ci.com/build/5032054518185984

**Comment by Apon77** [comment]
telegram id?

**Reply by mozzaru** [reply]
https://t.me/mozzaru

**Comment by Apon77** [comment]
can't be added because of user mismatch, and looking like fake identity

**Reply by mozzaru** [reply]
telegram and git names must be the same?

**Reply by mozzaru** [reply]
try again sir I've changed the telegram username and it's the same for both git and telegram
@Apon77
